src/Security/HelperV2/UserHelper.php

```diff
@@ -14,4 +15,12 @@ public function canManageBeneficiary(User $user, Beneficiaire $beneficiary): boo
             $beneficiary->getAffiliatedRelays()->toArray(),
         ));
     }
+
+    public function canManageProfessional(User $user, Membre $professional): bool
+    {
+        return 0 < count(array_intersect(
+            $user->getAffiliatedRelaysWithBeneficiaryManagement()->toArray(),
```

Comment:
I dont understant why you have to intersetc `AffiliatedRelays` and `AffiliatedRelaysWithCondition`

Reply:
We check that the connected user has at least one relay in common with tested professional